app/server/schemas.py

```diff
@@ -162,6 +162,7 @@ def filter_rejected(self, data, many):
     transfer_subtype        = fields.Function(lambda obj: obj.transfer_subtype.value)
     transfer_mode           = fields.Function(lambda obj: obj.transfer_mode.value)
     transfer_status         = fields.Function(lambda obj: obj.transfer_status.value)
+    blockchain_status       = fields.Function(lambda obj: obj.blockchain_status.value)
```

Comment:
Out of interest, can this schema just inherit from the blockchain taskable schema?